src/components/Composer/index.js

```diff
@@ -306,11 +306,18 @@ class Composer extends React.Component {
      * @param {ClipboardEvent} event
      */
     handlePaste(event) {
-        if (!this.props.checkComposerVisibility()) {
+        event.preventDefault();
+
```

Comment:
@robertKozik Why we moved this to the top?

Reply:
It causes a regression here, https://github.com/Expensify/App/issues/23502.

Reply:
My bad, sorry 😢

Reply:
Oh I think it's the same cause as this one https://github.com/Expensify/App/issues/23539

Reply:
It's the same cause, so I think one fix will fix both!